src/suites/cts/validation/error_scope.spec.ts

```diff
@@ -101,9 +107,14 @@ g.test('if an error scope matches an error it does not bubble to the parent scop
 g.test('if no error scope handles an error it fires an uncapturederror event', async t => {
   t.device.pushErrorScope('out-of-memory');
 
-  await t.expectUncapturedError(() => {
-    t.createErrorBuffer();
-  });
+  await t.expectUncapturedError(
```

Comment:
Then this would be

```
const uncapturedErrorEvent = await t.expectUncapturedError(() => {
  t.createErrorBuffer();
});
t.expect(uncapturedErrorEvent.error instanceof GPUValidationError);
```